### ISSUE TITLE
Training Collapsed

### ISSUE THREAD

**Issue by HowCCC** [microsoft/CADFusion#10]
Hello! Thks for your wonderful work! 
I am training the model locally using the code you privided, but the training loss will droped to zero early(during sl training stage), and the eval loss is Nan.
I am curious about what happend, I thought it maybe because of the CAD_dataset.py?

<img width="734" height="209" alt="Image" src="https://github.com/user-attachments/assets/d38f8303-6611-435f-ab4f-2e08a70ba1bb" />

**Comment by rwang37** [comment]
This did not happen during our training. May I ask for your python, pytorch and trl versions?

**Comment by HowCCC** [comment]
> This did not happen during our training. May I ask for your python, pytorch and trl versions?

Thanks for your reply, it works in my machine too. Thanks a lot.
